.gitignore

```diff
@@ -5,3 +5,4 @@ lerna-debug.log
 /dev
 /build
 .project.json
+.deployments.devnet.json
```

Comment:
I tried to load deployments by dynamic import, but it seems that webpack does not support this, so I added `.deployment.devnet.json` back, which is empty by default, if the contract is not deployed on the Internet, it will tell users to deploy contracts to the devnet first. WDYT?

Reply:
Sounds good to me!